problems/5.loops.js

```diff
@@ -1,33 +1,55 @@
 /** Returns every even number from 0 to a given number
-* @param {number} num - The number to iterate up to
-* @param {number[]} - An array of even numbers from 0 to {num}
-*/
-function evenNumbers() {
-
-}
+ * @param {number} num - The number to iterate up to
+ * @param {number[]} - An array of even numbers from 0 to {num}
+ */
+//function evenNumbers() {}
+const evenNumbers = (num) => {
+  let evenArray = [];
+  for (i = 0; i <= num; i += 2) {
+    if (i % 2 === 0) {
```

Comment:
Since your step is i += 2 there's no need to also check with the if

Reply:
I see it now.